### PR TITLE
(codestyle) Alphabetize ship names

### PR DESCRIPTION
**Pull request by a-random-lemurian**
This is primarily a "codestyle" pull request. This PR sorts names of the following phrases in A-Z order.

```yaml
- "republic capital"
- "free worlds capital"
- "syndicate capital"
- "militia"
- "republic small"
- "free worlds small"
```

This PR does not alter the C++ source code.

### CODE REVIEW

**Comment by tehhowch** [comment]
I'm not really interested in this change as-is, because it removes the existing local contexts (such as "mountain names") and replaces them with nothing.

If you really strongly want to modify these phrases, make sure the local context is kept. (One way would be to create new phrases that yield an item from that context.)

**Comment by a-random-lemurian** [comment]
> I'm not really interested in this change as-is, because it removes the existing local contexts (such as "mountain names") and replaces them with nothing.
> 
> If you really strongly want to modify these phrases, make sure the local context is kept. (One way would be to create new phrases that yield an item from that context.)

Could you explain how I would be able to do that? As in, would I add more names? Or would I add more phrases, as in, put the mountain names in phrases?

**Comment by Amazinite** [comment]
Something like this:
```
phrase "republic capital"
	word
		"R.N.S. "
	phrase
		"mountains"
		"rivers"
		<whatever else republic capital ships are named after>

phrase "mountains"
	<alphabetized list of mountains>

phrase "rivers"
	<alphabetized list of rivers>
```

**Comment by a-random-lemurian** [comment]
> Something like this:
> 
> ```
> phrase "republic capital"
> 	word
> 		"R.N.S. "
> 	phrase
> 		"mountains"
> 		"rivers"
> 		<whatever else republic capital ships are named after>
> 
> phrase "mountains"
> 	<alphabetized list of mountains>
> 
> phrase "rivers"
> 	<alphabetized list of rivers>
> ```

By the way, my knowledge of mountains and rivers suck. Are the river and mountain names mixed or are they separated, just with no comments that separate them?

**Comment by Amazinite** [comment]
Any progress being made here?

File: data/human/names.txt [hunk 1]
```diff
@@ -1935,172 +1935,172 @@ phrase "republic capital"
 	word
 		"R.N.S. "
 	word
```
Comment (LepRyot) [inline data/human/names.txt:1937]:
```suggestion
	phrase
		"mountains"

phrase "mountains"
```

File: data/human/names.txt [hunk 2]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
 		"I.M.S. "
 	word
```
Comment (LepRyot) [inline data/human/names.txt:2187]:
```suggestion
	phrase
		"trees"

phrase "trees"
```

File: data/human/names.txt [hunk 3]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
+		"Traveler"
 		"Walnut"
+		"Willow"
+		"Winterthorn"
+		"Yucca"
 
 phrase "free worlds capital"
 	word
 		"F.S. "
 	word
```
Comment (LepRyot) [inline data/human/names.txt:2273]:
```suggestion
	phrase "trees"
```

File: data/human/names.txt [hunk 4]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
+		"Traveler"
 		"Walnut"
+		"Willow"
+		"Winterthorn"
+		"Yucca"
 
 phrase "syndicate capital"
 	word
 		"S.S. "
 	word
```
Comment (LepRyot) [inline data/human/names.txt:2359]:
```suggestion
	phrase
		"roman emperor"
phrase "roman emperor"
```

File: data/human/names.txt [hunk 5]
```diff
@@ -2429,214 +2427,216 @@ phrase "syndicate capital"
...
-		"Nelson"
-		"Paraguay"
-		"Ural"
-		"Ganges"
-		"Oxus"
-		"Salween"
-		"Arkansas"
-		"Colorado"
-		"Dnieper"
+	word
```
Comment (LepRyot) [inline data/human/names.txt:2440]:
```suggestion
	phrase
		"river"
phrase "river"
```

File: data/human/names.txt [hunk 6]
```diff
@@ -2429,214 +2427,216 @@ phrase "syndicate capital"
...
+		"Yarra"
+		"Yenisei"
+		"Yukon"
+		"Zaire"
+		"Zambezi"
 
 phrase "free worlds small"
 	word
 		"F.S. "
 	word
```
Comment (LepRyot) [inline data/human/names.txt:2572]:
```suggestion
	phrase
		"flower"
phrase "flower"
```

File: data/human/names.txt [hunk 7]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
+		"Banyan"
+		"Baobab"
+		"Bartlett"
+		"Beech"
+		"Birch"
+		"Borage"
+		"Brisbane"
+		"Buckthorn"
+		"Carob"
 		"Cedar"
```
Comment (LepRyot) [inline data/human/names.txt:2289]: Duplicate list here
```suggestion
```

File: data/human/names.txt [hunk 8]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
+		"Coral"
+		"Cork"
+		"Cottonwood"
+		"Crabwood"
+		"Cypress"
+		"Dogwood"
 		"Douglas"
+		"Ebony"
+		"Elder"
 		"Elm"
```
Comment (LepRyot) [inline data/human/names.txt:2301]:
```suggestion
```

File: data/human/names.txt [hunk 9]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
-		"Cork"
-		"Hemlock"
-		"Larch"
-		"Sequoia"
-		"Redwood"
+		"Eucalyptus"
 		"Ginko"
```
Comment (LepRyot) [inline data/human/names.txt:2303]:
```suggestion
```

File: data/human/names.txt [hunk 10]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
-		"Dogwood"
-		"Bartlett"
-		"Magnolia"
-		"Baobab"
-		"Banyan"
-		"Cypress"
-		"Teak"
 		"Hawthorn"
```
Comment (LepRyot) [inline data/human/names.txt:2304]:
```suggestion
```

File: data/human/names.txt [hunk 11]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
-		"Poplar"
-		"Sycamore"
-		"Aspen"
-		"Holly"
-		"Ebony"
-		"Birch"
-		"Borage"
-		"Asphodel"
+		"Hazel"
 		"Heath"
```
Comment (LepRyot) [inline data/human/names.txt:2306]:
```suggestion
```

File: data/human/names.txt [hunk 12]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
+		"Larch"
 		"Laurel"
+		"Linden"
+		"Locust"
+		"Magnolia"
+		"Mangrove"
+		"Maple"
+		"Medlar"
+		"Mesquite"
 		"Myrtle"
```
Comment (LepRyot) [inline data/human/names.txt:2325]:
```suggestion
```

File: data/human/names.txt [hunk 13]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
-		"Yucca"
-		"Torrey"
-		"Sumac"
-		"Aloe"
-		"Alder"
-		"Hazel"
-		"Ironwood"
-		"Elder"
-		"Mangrove"
 		"Persimmon"
```
Comment (LepRyot) [inline data/human/names.txt:2327]:
```suggestion
```

File: data/human/names.txt [hunk 14]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
-		"Redbud"
-		"Coral"
-		"Locust"
-		"Hickory"
-		"Lancewood"
-		"Sassafras"
+		"Pinyon"
+		"Ponderosa"
+		"Poplar"
 		"Primrose"
```
Comment (LepRyot) [inline data/human/names.txt:2331]:
```suggestion
```

File: data/human/names.txt [hunk 15]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
+		"Tamarack"
+		"Tamarisk"
+		"Teak"
+		"Terebinth"
+		"Torrey"
+		"Traveler"
 		"Walnut"
+		"Willow"
+		"Winterthorn"
+		"Yucca"
```
Comment (LepRyot) [inline data/human/names.txt:2354]:
```suggestion
```

File: data/human/names.txt [hunk 16]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
-		"Cottonwood"
-		"Kapok"
-		"Mesquite"
+		"Raywood"
+		"Redbud"
+		"Redwood"
+		"Sassafras"
+		"Senna"
+		"Sequoia"
 		"Spruce"
```
Comment (LepRyot) [inline data/human/names.txt:2340]:
```suggestion
```

File: data/human/names.txt [hunk 17]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
-		"Sassafras"
+		"Pinyon"
+		"Ponderosa"
+		"Poplar"
 		"Primrose"
-		"Brisbane"
-		"Ash"
 		"Privet"
-		"Raywood"
 		"Quince"
```
Comment (LepRyot) [inline data/human/names.txt:2333]:
```suggestion
```

File: data/human/names.txt [hunk 18]
```diff
@@ -2185,190 +2185,188 @@ phrase "militia"
...
-		"Hickory"
-		"Lancewood"
-		"Sassafras"
+		"Pinyon"
+		"Ponderosa"
+		"Poplar"
 		"Primrose"
-		"Brisbane"
-		"Ash"
 		"Privet"
```
Comment (LepRyot) [inline data/human/names.txt:2332]:
```suggestion
```

**Review by LepRyot — CHANGES_REQUESTED** [review]
It doesn't look like they're mixed; each one of the ship name lists references a unique set of nouns

**Comment by a-random-lemurian** [comment]
@LepRyot LGTM, let me commit and clean up all those junk commits that might be created

**Comment by a-random-lemurian** [comment]
@Amazinite Should be good now, yes?

**Pull request merged**